contracts/interfaces/IDCAStrategies.sol

```diff
@@ -6,19 +6,40 @@ import '@mean-finance/dca-v2-core/contracts/interfaces/IDCAHub.sol';
 import '@mean-finance/nft-descriptors/solidity/interfaces/IDCAHubPositionDescriptor.sol';
 
 interface IDCAStrategiesManagementHandler {
+  /**
+   * @notice Emitted when a new strategy is created
+   * @param strategyId The id of the new strategy
+   * @param strategy The struct with the values of the new strategy
+   */
+  event StrategyCreated(uint80 strategyId, Strategy strategy, IDCAStrategies.ShareOfToken tokens);
+
+  /// @notice Thrown when strategy name is more than 32 bytes
+  error NameTooLong();
+
+  /// @notice Thrown when a provided array is empty
+  error LengthZero();
```

Comment:
When are you emitting this?

Reply:
It was on some piece of code I deleted, but I will use it `_checkTokenSharesSanity()`